spqlios/arithmetic/vec_znx_arithmetic.h

```diff
@@ -34,27 +34,54 @@ typedef struct cnv_pvec_l_t CNV_PVEC_L;
 /** @brief opaque type that represents a prepared right convolution vector product */
 typedef struct cnv_pvec_r_t CNV_PVEC_R;
 
-/** @brief allocates a prepared matrix (release with free) */
-EXPORT VMP_PMAT* vmp_pmat_alloc(const MODULE* module,           // N
-                                uint64_t nrows, uint64_t ncols  // dimensions
-);
+EXPORT void spqlios_free(void* address);
 
-/** @brief allocates a vec_znx in DFT space (release with free) */
-EXPORT VEC_ZNX_DFT* vec_znx_dft_alloc(const MODULE* module,  // N
-                                      uint64_t size);
+EXPORT void* spqlios_alloc(uint64_t size);
 
-/** @brief allocates a vec_znx_big (release with free) */
-EXPORT VEC_ZNX_BIG* vec_znx_big_alloc(const MODULE* module,  // N
-                                      uint64_t size);
+EXPORT void spqlios_debug_free(void* address);
```

Comment:
these two _debug_alloc and _debug_free functions should only be declared in commons-private.h (they can also be static in commons.c, and used only in the implem of spqlios_alloc and spqlios_free)